sandbox/src/mocks/getReferralRequest/responses/Unbooked.json

```diff
@@ -0,0 +1,80 @@
+{
+  "id": "000000070000",
+  "meta": {
+    "versionId": "3",
+    "profile": [
+      "http://fhir.nhs.uk/StructureDefinition/ers-referralrequest-1"
+    ]
+  },
+  "resourceType": "ReferralRequest",
+  "contained": [
+    {
+      "id": "Patient-9912003888",
+      "meta": {
+        "profile": [
+          "http://fhir.nhs.uk/StructureDefinition/ers-patient-1"
+        ]
+      },
+      "resourceType": "Patient",
+      "identifier": [
+        {
+          "system": "http://fhir.nhs.uk/Id/nhs-number",
+          "value": "9912003888"
+        }
+      ]
+    },
+    {
+      "id": "DocumentReference-70002",
+      "meta": {
+        "profile": [
+          "http://fhir.nhs.uk/StructureDefinition/ers-documentreference-1"
+        ]
+      },
+      "resourceType": "DocumentReference",
+      "type": {
+        "coding": [
+          {
+            "system": "http://fhir.nhs.uk/ValueSet/ers-attachmenttype-1",
+            "code": "REFERRER"
+          }
+        ]
+      },
+      "status": "current",
+      "indexed": "ignoreDate",
+      "description": "",
```

Comment:
Do we need a more realistic values for this description (again, I just ripped this from the XAPI IT)